### DISCUSSION TITLE
Tile Grid system for scratch.

### DISCUSSION THREAD

**Discussion by V-boy255** [ScratchAddons/ScratchAddons#2250] [Ideas]
It could be good to make a tile grid system on scratch to make levels easily for our games, like in unity, godot...
![tilemap](https://user-images.githubusercontent.com/82961362/115582798-952fdc00-a2c9-11eb-8e4f-052fcba4c991.png)

**Comment by cobaltt7** [comment]
In the paint editor?

**Comment by BlueDome77** [comment]
It would need to have it's own section as it would be difficult to implement it in the paint editor

**Comment by Endermaster3690** [comment]
probably related: #5219
